ocs_ci/ocs/constants.py

```diff
@@ -1154,7 +1218,7 @@
 SCALEUP_TERRAFORM_DATA_DIR = "scaleup_terraform_data"
 SCALEUP_VSPHERE_DIR = os.path.join(
     EXTERNAL_DIR,
-    f"v4-scaleup/ocp4-rhel-scaleup/aos-{get_ocp_version(seperator='_')}/vsphere",
+    # f"v4-scaleup/ocp4-rhel-scaleup/aos-{get_ocp_version(seperator='_')}/vsphere",
```

Comment:
Is this change related to this PR?

Reply:
accidently left this. Thanks.

This should be removed by Vijay https://github.com/red-hat-storage/ocs-ci/issues/9432 it creates circular import when exporting any OCP related class to python console